### PR TITLE
fix(e2ei): revoked certificates not updated WPB-3230

### PR DESCRIPTION
**Pull request by David-Henner**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-3230" title="WPB-3230" target="_blank">StoryWPB-3230</a>  [iOS] 5.3.18 Client with revoked certificate
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

I do not see when users have revoked certificates (including my own user). Revoked certificates continue to show as valid, unless I do a fresh login.

### Causes

Due to caching, we don't pass the regenerated CRL containing the newly revoked certificate to CC. Instead we pass the old cached CRL.

### Solutions

Use an `URLSession` with an `ephemeral` configuration and disable caching

**Note:** I'm also making `updateAllStatuses` non-throwing to avoid returning early in case we fail to update the status of one of the conversation

### CODE REVIEW

File: wire-ios-data-model/Source/E2EIdentity/MLSConversationVerificationStatusUpdater.swift [hunk 1]
```diff
@@ -44,7 +44,9 @@ public class MLSConversationVerificationStatusUpdater: MLSConversationVerificati
 
     // MARK: - Public interface
 
-    public func updateAllStatuses() async throws {
+    public func updateAllStatuses() async {
+        WireLogger.ear.info("updating all MLS conversations verification status")
```
Comment (netbe) [inline wire-ios-data-model/Source/E2EIdentity/MLSConversationVerificationStatusUpdater.swift]: is it the right logger?

**Review by netbe — APPROVED** [review]
just logger to be changed

File: wire-ios-data-model/Source/E2EIdentity/MLSConversationVerificationStatusUpdater.swift (hunk 1 above)
Comment (David-Henner) [reply wire-ios-data-model/Source/E2EIdentity/MLSConversationVerificationStatusUpdater.swift]: which logger would you use instead?

Comment (David-Henner) [reply wire-ios-data-model/Source/E2EIdentity/MLSConversationVerificationStatusUpdater.swift]: ah no I see my mistake

Comment (David-Henner) [reply wire-ios-data-model/Source/E2EIdentity/MLSConversationVerificationStatusUpdater.swift]: good catch

**Review by typfel — APPROVED** [review]

**Pull request merged**